flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/schema/FlinkRelOptTable.scala

```diff
@@ -221,12 +230,40 @@ class FlinkRelOptTable protected(
     val cluster: RelOptCluster = context.getCluster
     if (table.isInstanceOf[TranslatableTable]) {
       table.asInstanceOf[TranslatableTable].toRel(context, this)
-    } else if (Hook.ENABLE_BINDABLE.get(false)) {
```

Comment:
why deleting these?

Reply:
Because Flink never use it, the orgina `ENABLE_BINDABLE` is only used for Calcite bindables. The code is wrong.